### DISCUSSION TITLE
Easier ability to toggle between assistants

### DISCUSSION THREAD

**Discussion by danielrosehill** [danny-avila/LibreChat#5104] [Feature Requests & Suggestions]
Hey!

A quick feature request. 

When accessing Libre chat through the mobile it's tricky to switch between the assistants. In fact it's also quite challenging from the desktop UI.

If there was something like a drop-down that allied the user to easily switch between the assistants that are configured and available for that account it would make the utility a lot easier to use.

If it helps with the explanation a couple of screenshots I took when accessing my assistants from mobile:

![Screenshot_20241225_170641_Chrome.jpg](https://github.com/user-attachments/assets/a942262f-7c6c-4187-bf49-ced6ad40bd69)

![Screenshot_20241225_170604_Chrome.jpg](https://github.com/user-attachments/assets/093b1c9e-b7b6-457a-b81a-bcd9859dffa4)

**Comment by danny-avila** [comment]
Thanks for the feedback! One quick way is to use @ (mention) at the start of chat input. I need to advertise this better in the ui/ux
